lib/rajska.ex

```diff
@@ -99,13 +99,15 @@ defmodule Rajska do
       def is_field_authorized?(nil, _scope_by, _source), do: false
       def is_field_authorized?(%{id: user_id}, scope_by, source), do: user_id === Map.get(source, scope_by)
 
-      def has_user_access?(%user_struct{id: user_id} = current_user, scoped_struct, field_value) do
+      def has_user_access?(%user_struct{id: user_id} = current_user, scoped_struct, field_value, nil) do
         is_super_user? = current_user |> get_user_role() |> is_super_role?()
         is_owner? = (user_struct === scoped_struct) && (user_id === field_value)
 
         is_super_user? || is_owner?
       end
 
+      def has_user_access?(_, _, _, rule), do: raise "Rule :#{rule} not defined in Authorization module"
```

Comment:
This line is not needed, since this error is pretty much the same error that elixir will raise